types/request-promise/request-promise-tests.ts

```diff
@@ -40,7 +40,15 @@ rp('http://google.com').promise().then(console.dir);
 //rp('http://google.com').promise().bind(this).then(console.dir);
 
 rp({ uri: 'http://google.com', resolveWithFullResponse: true }).then((response) => {});
-rp({ uri: 'http://google.com', simple: false }).catch((reason) => {});
+
+rp({ uri: 'http://google.com', simple: false }
+).catch(errors.StatusCodeError, (reason: errors.StatusCodeError) => {
```

Comment:
Why do you pass two arguments to the `catch` method?

Reply:
Oops! Fixed it. I copy-pasted those lines from below. Thanks for catching this.